### PR TITLE
Add styling UI

### PR DESCRIPTION
**Pull request by bennisan86**

### CODE REVIEW

File: utils/theme.ts [hunk 1]
```diff
@@ -95,15 +97,15 @@ const FormLabel = {
...
-    variant: 'secondary',
+    variant: 'primary',
   },
   variants: {
     primary: {
-      fontFamily: `'Orbitron', sans-serif`,
-    },
-    secondary: {
-      fontFamily: `'Press Start 2P', cursive`,
+      fontFamily: `'Unbounded', sans-serif`,
```
Comment (ECWireless) [inline utils/theme.ts:108]: "Unbounded" seems seems great as a body font, but feels a little lacking as a heading in giving people a vague-ish impression that they are playing a video game. Wondering if there's a font out there that gives the same feeling as "Press Start 2P," but isn't as aggressive or ineligible

File: components/GameCard.tsx [hunk 2]
```diff
@@ -21,79 +20,69 @@ export const GameCard: React.FC<GameCardProps> = ({
 }) => {
   return (
     <Box
-      border="3px solid black"
-      borderBottom="5px solid black"
-      borderRight="5px solid black"
-      p={8}
+      p={0}
```
Comment (ECWireless) [inline components/GameCard.tsx:23]: I think this defaults to `0`, so can be removed

File: components/GameCard.tsx [hunk 3]
```diff
@@ -21,79 +20,69 @@ export const GameCard: React.FC<GameCardProps> = ({
...
-            transition="all 0.3s ease"
-            variant="primary"
+            fontSize="40px"
+            fontWeight="normal"
+            lineHeight="40px"
           >
             {name}
           </Heading>
-          <Text fontSize="sm">{shortenText(description, 60)}</Text>
+          {/* <Text fontSize="sm">{shortenText(description, 60)}</Text> */}
```
Comment (ECWireless) [inline components/GameCard.tsx:57]: I'm personally against removing the description for games. I imagine people will want to understand at a high-level what the game is about without having to click into each one

File: pages/all-games.tsx [hunk 4]
```diff
@@ -26,16 +26,19 @@ export default function AllGames(): JSX.Element {
...
-            key={game.id}
-            chainId={chain?.id ?? DEFAULT_CHAIN.id}
-            {...game}
-          />
-        ))}
-      </Flex>
-    </VStack>
+    <Box pt={24} pb={20} pl={'10vw'}>
+      {/* <Text opacity={0.5} mb={10}>ALL GAMES :</Text> */}
+      <VStack as="main">
```
Comment (ECWireless) [inline pages/all-games.tsx:31]: Would remove this unnecessary div

File: utils/theme.ts [hunk 5]
```diff
@@ -3,87 +3,89 @@ import { css } from '@emotion/react';
...
+      pb: '2',
+      px: '6',
     },
     md: {
       fontSize: '16px',
-      h: '50px',
-      w: '180px',
+      lineHeight: '20px',
+      h: 'auto',
+      pt: '2',
```
Comment (ECWireless) [inline utils/theme.ts:49]: Should this be `pt: 3`? Looks slightly off-center:

Screenshot 2023-10-11 at 3 24 45 PM

**Review by ECWireless — APPROVED** [review]
Home page looks great! My main high-level feedback is that the heading font seems a little too dull to give the subtle impression of playing a video game (although the original font was a little too aggressive in that direction; hoping we can find something in the middle that works). Also, realizing we should probably make sure class tags allow GameMaster uploaded class "emblems" to be rendered somewhere (maybe to the left of the text?).

Anyway, left a few comments, most of which I can fix now!

File: components/ActiveLink.tsx [hunk 6]
```diff
@@ -9,12 +9,16 @@ export const ActiveLink: React.FC<LinkProps> = ({ href, ...props }) => {
   return (
     <Link
       as={NextLink}
-      borderBottom={isActive ? '2px solid black' : '2px solid transparent'}
       href={href ?? ''}
-      _hover={{ borderBottom: '2px solid black' }}
+      fontWeight="medium"
+      fontSize={'sm'}
```
[the remark this answers is not in the document]
Comment (ECWireless) [reply components/ActiveLink.tsx]: Done in e0ba468361fa5bc11fcc9763978243a3f891fed1

File: components/ActiveLink.tsx [hunk 7]
```diff
@@ -9,12 +9,16 @@ export const ActiveLink: React.FC<LinkProps> = ({ href, ...props }) => {
...
+      color={isActive ? 'accent' : 'inherit'}
+      w="110px"
```
[the remark this answers is not in the document]
Comment (ECWireless) [reply components/ActiveLink.tsx]: Done in e0ba468361fa5bc11fcc9763978243a3f891fed1

File: components/Layout.tsx [hunk 8]
```diff
@@ -17,34 +18,35 @@ export const Layout: React.FC<{ children: JSX.Element }> = ({ children }) => {
...
+            as={NextLink}
+            ml={5}
+            href="/"
+            _hover={{ textDecoration: 'none', color: 'accent' }}
+          >
+            <Heading
+              fontSize="22px"
+              fontWeight="regular"
+              textTransform="uppercase"
+              color={'inherit'}
```
[the remark this answers is not in the document]
Comment (ECWireless) [reply components/Layout.tsx]: Fixed in e0ba468361fa5bc11fcc9763978243a3f891fed1

File: components/Layout.tsx [hunk 9]
```diff
@@ -17,34 +18,35 @@ export const Layout: React.FC<{ children: JSX.Element }> = ({ children }) => {
...
+        <Flex align="center" h="100%" justify="start" w="100%">
+          <Link
+            as={NextLink}
+            ml={5}
+            href="/"
+            _hover={{ textDecoration: 'none', color: 'accent' }}
+          >
+            <Heading
+              fontSize="22px"
+              fontWeight="regular"
```
[the remark this answers is not in the document]
Comment (ECWireless) [reply components/Layout.tsx]: Removed in e0ba468361fa5bc11fcc9763978243a3f891fed1

File: pages/index.tsx [hunk 10]
```diff
@@ -8,26 +18,146 @@ export default function Home(): JSX.Element {
...
+      px="10vw"
+      backgroundImage="url('/RG_CS_bg.png')"
+      backgroundPosition="0 -20vh"
+      backgroundRepeat="no-repeat"
+      backgroundSize="100%"
+    >
+      <Heading
+        fontSize="66px"
+        lineHeight="56px"
+        fontWeight={'regular'}
```
[the remark this answers is not in the document]
Comment (ECWireless) [reply pages/index.tsx]: Remove in e0ba468361fa5bc11fcc9763978243a3f891fed1

File: pages/index.tsx [hunk 11]
```diff
@@ -8,26 +18,146 @@ export default function Home(): JSX.Element {
...
+        color={'whiteAlpha.700'}
+        mt={20}
+        mb={6}
+        fontSize={'14px'}
+        textTransform={'uppercase'}
+      >
+        Most recent :
+      </Text>
+
+      {/* This could be Charactercard? */}
```
[the remark this answers is not in the document]
Comment (ECWireless) [reply pages/index.tsx]: Added in e0ba468361fa5bc11fcc9763978243a3f891fed1

File: pages/index.tsx [hunk 12]
```diff
@@ -8,26 +18,146 @@ export default function Home(): JSX.Element {
...
+          <Heading _hover={{ color: 'accent', cursor: 'pointer' }}>
+            McLizard the Hizard
+          </Heading>
+          <Link fontSize="sm" href={`/`} isExternal fontWeight={300}>
+            0xIIO...mlmz
+          </Link>
+          <HStack spacing={4} mt={6} mb={8} flexWrap={'wrap'}>
+            {/* Maybe this is a Classtag? */}
+            <HStack spacing={0}>
+              <Box h={'22px'} w={'6px'} bg={'softgreen'} m={0}></Box>
```
[the remark this answers is not in the document]
Comment (ECWireless) [reply pages/index.tsx]: Removed in e0ba468361fa5bc11fcc9763978243a3f891fed1

File: pages/index.tsx [hunk 13]
```diff
@@ -8,26 +18,146 @@ export default function Home(): JSX.Element {
...
+                bg={'softgreen'}
+                color={'dark'}
+                fontWeight={'bold'}
+                fontSize={'sm'}
+                py={2}
+                px={8}
+              >
+                Wizard
+              </Text>
+              <Box h={'22px'} w={'6px'} bg={'softgreen'}></Box>
```
[the remark this answers is not in the document]
Comment (ECWireless) [reply pages/index.tsx]: Done in e0ba468361fa5bc11fcc9763978243a3f891fed1

File: components/GameCard.tsx (hunk 2 above)
Comment (ECWireless) [reply components/GameCard.tsx]: Removed in e0ba468361fa5bc11fcc9763978243a3f891fed1

File: pages/all-games.tsx (hunk 4 above)
Comment (ECWireless) [reply pages/all-games.tsx]: Removed in e0ba468361fa5bc11fcc9763978243a3f891fed1

File: utils/theme.ts (hunk 1 above)
Comment (bennisan86) [reply utils/theme.ts:107]: The thing is that the font Unbounded wasn't loading * *facepalm*, so what you were seeing wasn't as intended. See the difference below. I really hope you can reconsider and get on board with this font, bcs I think it is pretty perfect for what we're trying to do in this design. 

**What you saw:**
Screenshot 2023-10-12 at 09 25 00

**What it should be (and is now, I fixed the google fonts):**
Screenshot 2023-10-12 at 09 31 40

File: components/GameCard.tsx (hunk 3 above)
Comment (bennisan86) [reply components/GameCard.tsx:57]: k, cool, will give it a good place on the card!

File: utils/theme.ts (hunk 5 above)
Comment (bennisan86) [reply utils/theme.ts:48]: Yeah again an issue related to the font not loading. See below, the button with Unbounded, seems to fix the off center thing? Or what do you think @ECWireless ?
Screenshot 2023-10-12 at 09 39 27

**Pull request merged**